src/components/JumbotronVideoComponent.vue

```diff
@@ -9,7 +9,13 @@ const props = defineProps<{
   /**
    * If set, the main element will have a background video.
    */
-  video?: string;
+  videos?: string[];
```

Comment:
Here, you have two features: adding multiple videos and the split-screen layout. This should go into its own feature!